src/Admin/EmailTemplateAdmin.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+namespace App\Admin;
+
+use App\Entity\Geo\Zone;
+use App\Entity\Scope;
+use App\Form\Admin\UnlayerContentType;
+use Doctrine\ORM\QueryBuilder;
+use Sonata\AdminBundle\Admin\AdminInterface;
+use Sonata\AdminBundle\Datagrid\DatagridMapper;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Datagrid\ProxyQueryInterface;
+use Sonata\AdminBundle\Filter\Model\FilterData;
+use Sonata\AdminBundle\Form\FormMapper;
+use Sonata\AdminBundle\Form\Type\ModelAutocompleteType;
+use Sonata\DoctrineORMAdminBundle\Datagrid\ProxyQuery;
+use Sonata\DoctrineORMAdminBundle\Filter\CallbackFilter;
+use Sonata\DoctrineORMAdminBundle\Filter\ModelFilter;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+
+class EmailTemplateAdmin extends AbstractAdmin
+{
+    private int $emailTemplateUnlayerTemplateId;
+
+    private DataTransformerInterface $dataTransformer;
+
+    public function __construct(
+        string $code,
+        string $class,
+        string $baseControllerName,
+        string $emailTemplateUnlayerTemplateId,
+        DataTransformerInterface $dataTransformer
+    ) {
+        parent::__construct($code, $class, $baseControllerName);
+
+        $this->emailTemplateUnlayerTemplateId = (int) $emailTemplateUnlayerTemplateId;
+        $this->dataTransformer = $dataTransformer;
+    }
+
+    protected function configureDatagridFilters(DatagridMapper $datagridMapper): void
+    {
+        $datagridMapper
+            ->add('label', null, [
+                'label' => 'Label',
+                'show_filter' => true,
+            ])
+            ->add('scopes', CallbackFilter::class, [
+                'show_filter' => true,
+                'field_type' => EntityType::class,
+                'field_options' => [
+                    'class' => Scope::class,
+                ],
+                'callback' => function (ProxyQuery $qb, string $alias, string $field, FilterData $value) {
+                    if (!$value->hasValue()) {
+                        return false;
+                    }
+
+                    $qb
+                        ->andWhere(sprintf('%1$s.scopes IS NOT NULL AND FIND_IN_SET(:scope, %1$s.scopes) > 0', $alias))
```

Comment:
I think we can avoid null check here :

```suggestion
                        ->andWhere(sprintf('FIND_IN_SET(:scope, %s.scopes) > 0', $alias))
```